docs/release-notes/index.md

```diff
@@ -0,0 +1,1243 @@
+---
+navigation_title: "Elastic APM Java Agent"
```

Comment:
Every markdown in the `release-notes` folder has the same `Elastic APM Java Agent` title:
![image](https://github.com/user-attachments/assets/4fbf9f86-7e16-4eed-814b-7431a8589e85)

Reply:
This is expected for now. It will be resolved when we're able to bundle pages from all repos together. https://github.com/elastic/docs-builder/issues/621